pkg/deployments/tasks/20221111-authorizer-adaptor-entrypoint/readme.md

```diff
@@ -0,0 +1,11 @@
+# 2022-11-11 - Authorizer Adaptor Entrypoint
+
+Deployment of the `AuthorizerAdaptorEntrypoint`, a gateway contract for the `AuthorizerAdaptor` that ensures correct interactions with the adaptor and the system of permissions behind it.
```

Comment:
```suggestion
Deployment of the `AuthorizerAdaptorEntrypoint`, a gateway contract created to address a critical bug found in the `AuthorizerAdaptor` that could lead to unintended escalation of prvileges. This contract ensures correct interactions with the adaptor and the system of permissions behind it by working in combination with `TimelockAuthorizer` so that all Adaptor calls made through the Entrypoint check for permissions correctly, while any not made via the Entrypoint fail.
```
Maybe expand this to reflect the new docs in the contracts